pkg/kudoctl/util/repo/repo.go

```diff
@@ -0,0 +1,52 @@
+package repo
+
+import (
+	"github.com/kudobuilder/kudo/pkg/apis/kudo/v1alpha1"
+)
+
+// Entry represents a collection of parameters for framework repository
+type Entry struct {
+	Name      string `json:"name"`
+	LocalPath string `json:"localPath"`
+	URL       string `json:"url"`
+}
+
+// Metadata for a Framework. This models the structure of a bundle.yaml file.
+type Metadata struct {
```

Comment:
how is the metadata file related to the metadata we'll be storing also in FV? https://github.com/kudobuilder/kudo/pull/277

Reply:
For me the metadata described in #277 is within our Framework versus this Metadata solely defines the one within a repository 🤔 So what this should be here is how from a repository perspective the actual `index.yaml` describes the Framework and should look like, and not how the Framework structs will look like because of KEP-0009.

Whats living within the `.tgz` should be KEP-0009 and not important to KEP-0010, then the structure within the `.tgz` becomes almost irrelevant here.

Though it shares the same name Metadata it would be better described as "Metadata of Repository Framework" vs. "Metadata within a Framework"?